pbj-core/pbj-grpc-helidon/docs/design.md

```diff
@@ -0,0 +1,450 @@
+# Architectural Design
```

Comment:
Are there any plans to support [gRPC Reflection](https://grpc.io/docs/guides/reflection/) to promote discovery?  Java Greeter Reflection [example](https://github.com/grpc/grpc-java/tree/master/examples/example-reflection).  It looks like there was [support in Helidon v3](https://helidon.io/docs/v3/apidocs/io.helidon.grpc.server/io/helidon/grpc/server/ProtoReflectionService.html) but I don't see the same class in v4 at this time.

Reply:
Great question! I think we should add it into the "Future Work" section.